test/buildah-bud/README.md

```diff
@@ -1,5 +1,4 @@
-buildah-bud tests under podman
-==============================
+# buildah-bud tests under podman
```

Comment:
Review recommendation for this file: '...' menu (top right) -> View file, instead of trying to read the diffs